src/spawn.c

```diff
@@ -67,6 +67,7 @@
 #else
 # include "support.h"
 #endif
+#include "utils.h"
```

Comment:
use of the `utils_*()` functions will break the "standalone-ness" of the spawn code.  It's probably alright for now, but might be annoying if we want to use it for implementing a run program wrapper.